ocrd_tesserocr/common.py

```diff
@@ -83,16 +195,25 @@ def image_from_page(workspace, page,
         page_image = workspace.resolve_image_as_pil(
             alternative_image[-1].get_filename())
     elif border:
```

Comment:
I do not fully understand the logistics here: What if we have `border` without `Coords`? (See `if` condition above).

Reply:
Technically, `BorderType` needs to have a `Coords` (`minoccurs=1`), and `CoordsType` needs to have a `@points` (`required=True`). But the input PAGE could be invalid, and then yes, this would lead to entering line 85 without `page_polygon`... I will fix.

Reply:
No I won't. If the PAGE is invalid, an exception is inevitable anyway.